lib/mixin/Mixin.hx

```diff
@@ -225,6 +226,12 @@ class Mixin
 		var classFql = getFqlClassName(lc);				
 		var fields = Context.getBuildFields();
 		var cached = mixins.get(mixinFql);
+
+		#if display
+		if(null == cached) {
```

Comment:
The display server constantly crashed due to the cached variable being `null` for some reason. I have no idea why, but this poor fixes made the issue disappear

Reply:
Some time display server acts weird with complex macro, it's ok